Tasks/Src/Task_Init.c

```diff
@@ -86,6 +87,14 @@ void Task_Init(void *p_arg) {
             TASK_CLI_STACK_SIZE,		        // Stack size
             );
     */
+    RTOS_BPS_TaskCreate(&Print_TCB,
```

Comment:
maybe a suggestion but theoretically we should only be doing buffer print when actively debugging, so maybe we can only define this task when some debug flag is set. This is optional though and doesn't matter too much if it adds complexity as the task is just above idle task in priority so its prob aight